password/test/tasty/Scrypt.hs

```diff
@@ -11,9 +11,10 @@ import Test.Tasty.Golden (goldenVsString)
 import Test.Tasty.QuickCheck
 import Test.QuickCheck.Instances.Text ()
 
-#ifndef IS_MAC_OS
+#ifndef darwin_HOST_OS
 import qualified Crypto.Scrypt as Scrypt
 #endif
```

Comment:
GHC provides os-specific CPP flags, so we don't need to define our own:

https://guide.aelve.com/haskell/cpp-vww0qd72#item-qbrikka7

Reply:
Dang I needed something like this, I was looking all over for the OS CPP flag